h/api/storage.py

```diff
@@ -33,15 +34,15 @@ def annotation_from_dict(data):
     return models.elastic.Annotation(data)
 
 
-def fetch_annotation(request, id, _postgres=None):
+def fetch_annotation(request, annotation_id, _postgres=None):
```

Comment:
[Done](https://github.com/hypothesis/h/pull/3256/commits/9645f6351ab37ebccd883e8deca48a0992878c6a) by using `id_` instead of `annotation_id`